### ISSUE TITLE
Inconsistent background & mesh for 2D and 3D graphs

### ISSUE THREAD

**Issue by pepe78** [matplotlib/matplotlib#20352]
https://github.com/pepe78/model-blending

2d graph - default white background & no mesh
3d graph - default gray background & mesh

inconsistent...

**Comment by jklymak** [comment]
I personally think thats fine for the different types of display requirements.

**Comment by pepe78** [comment]
If you do, feel free to close this. I would prefer same default style for both.

**Comment by WeatherGod** [comment]
I am curious how you think it should look? I am open to seeing improved
default styles, but a strong case needs to be made to make any such changes.

Right now, the rationale is that the 3D plotting area needs to look like a
3D figure region that can be interacted with, and the grid lines and "box"
help to create that effect. If we made grid lines go away, and make the
facecolors white, I don't think it would look like a 3D plot anymore, and
I'd think one would lose the sense of orientation for interactivity. I'd be
happy to be proven wrong, though, as the choice of the grid lines and face
colors is largely a legacy one.

**Comment by pepe78** [comment]
My personal preference would be grid & white (transparent) background. So grid from default 3d view, white background from 2d default view for both 2d & 3d. Maybe you guys should do survey from graphical designers though :) and not listen to me... As I said, feel free to close this if you guys are happy with your current design.

**Comment by timhoffm** [comment]
We are very conservative with default style changes, because that'll affect the plots of all users. A default style change would be a concerted action, possibly changing multple things, and probably be backed up by UX graphics design research before. I've added this to the meta-ticket on future style changes #14331 so that it's not forgotton once we set out for a new design, but don't expect anything soon.
